src/pages/PageCallManage/PageCallManage.tsx

```diff
@@ -0,0 +1,272 @@
+              {
+                label: this.showButtonsInVideoCall
+                  ? intl`Hide call menu buttons`
+                  : intl`Show call menu buttons`,
+                onPress: this.toggleButtons,
+              },
+            ]
+          : undefined
+      }
+      noScroll
+      onBack={Nav().backToPageCallRecents}
+      title={c?.partyNumber || intl`Connection failed`}
+      transparent={!c?.transferring}
+    >
+      {!c ? null : c.transferring ? (
+        <PageTransferAttend />
+      ) : (
+        <>
+          {isVideoEnabled && this.renderVideo(c)}
+
+          {!isVideoEnabled && (
+            <CallerInfo
+              isUserCalling={!c.partyNumber.includes('+')}
+              callerName={c.getCallerName}
+              callerNumber={c.partyNumber}
+              containerStyle={{ marginTop: c.getCallerName ? '5%' : '20%' }}
+            />
+          )}
+          {this.renderCallTime()}
+          {this.renderBtns(c, isVideoEnabled)}
+          {this.renderHangupBtn(c, isVideoEnabled)}
+        </>
+      )}
+    </Layout>
+  )
+
+  renderVideo = (c: Call) => (
+    <>
+      <View style={styles.videoSpace} />
+      <View style={styles.video}>
+        <VideoPlayer sourceObject={c.remoteVideoStreamObject} />
+      </View>
+      <RnTouchableOpacity
+        onPress={this.toggleButtons}
+        style={StyleSheet.absoluteFill}
+      />
+    </>
+  )
+
+  renderBtns = (c: Call, isVideoEnabled?: boolean) => {
+    if (isVideoEnabled && !this.showButtonsInVideoCall) {
+      return null
+    }
+    const Container = isVideoEnabled ? RnTouchableOpacity : View
+    const activeColor = CustomColors.IconActiveBlue
+    const n = callStore.backgroundCalls.length
+    return (
+      <Container
+        onPress={isVideoEnabled ? this.toggleButtons : undefined}
+        style={[styles.btns, isVideoEnabled && styles.btnsIsVideoEnabled]}
+      >
+        <View style={styles.btnsVerticalMargin} />
+        {/* TODO add Connecting... */}
+        <View style={!c.answered && styles.btnsHidden}>
+          <View style={styles.btnsInnerView}>
+            <CallActionButton
+              bgcolor={c.muted ? activeColor : 'white'}
+              name={c.muted ? intl`Unmute` : intl`Mute`}
+              onPress={() => c.toggleMuted()}
+              textcolor={c.muted ? 'white' : 'black'}
+              image={
+                c.muted ? CustomImages.MicrophoneOff : CustomImages.Microphone
+              }
+              imageStyle={styles.actionBtnImage}
+            />
+            <CallActionButton
+              bgcolor={c.holding ? activeColor : 'white'}
+              name={c.holding ? intl`Unhold` : intl`Hold`}
+              onPress={() => c.toggleHold()}
+              textcolor={c.holding ? 'white' : 'black'}
+              image={c.holding ? CustomImages.Unhold : CustomImages.Pause}
+              imageStyle={styles.actionBtnImage}
+            />
+            <CallActionButton
+              bgcolor={'white'}
```

Comment:
declare colors in specific file and use from there

Reply:
I am modifying this file for the next ticket, It will be fixed in the next PR.